### PR TITLE
Show ips based on a flag value

### PR DESCRIPTION
**Pull request by hub-cap**
- Added a flag value to determine whether to show the ip
- Added the logic to pass to the views to show the ip or not
- Added initial db creation in prepare

### CODE REVIEW

**Comment by cp16net** [comment]
+1

**Comment by TimSimpsonR** [comment]
+1

File: reddwarf/instance/models.py [hunk 1]
```diff
@@ -133,7 +154,9 @@ def create(cls, context, name, flavor_ref, image_id):
             status=ServiceStatuses.NEW)
         # Now wait for the response from the create to do additional work
         guest = create_guest_client(context, db_info.id)
-        guest.prepare(512, [])
+        # populate the databases
+        model_schemas = populate_databases(databases)
+        guest.prepare(512, model_schemas)
```
Comment (rnirmal) [inline reddwarf/instance/models.py:159]: Why is the memory hardcoded... I suppose we were waiting for flavors. Might be good to add it now.

Comment (hub-cap) [inline reddwarf/instance/models.py:159]: Tim said he was working on that. I'm not touching it just to muck w his code :)

File: reddwarf/instance/views.py [hunk 2]
```diff
@@ -16,18 +16,29 @@
 #    under the License.
 
 
+def get_ip_address(addresses):
```
Comment (rnirmal) [inline reddwarf/instance/views.py:19]: Hmmm... I think if we are returning ips we should return all the ips not just the one private. And go with either {'ips': []} or {'addresses': [{'ip': ''}]}

Comment (hub-cap) [inline reddwarf/instance/views.py:19]: K I can do that. Maybe just addresses w a list?

File: reddwarf/instance/models.py (hunk 1 above)
Comment (rnirmal) [inline reddwarf/instance/models.py:159]: k just wanted to make sure :)

File: reddwarf/instance/views.py (hunk 2 above)
Comment (rnirmal) [inline reddwarf/instance/views.py:19]: Yeah just a list should do. I don't think we want to think about public vs private for now.

**Comment by ed-** [comment]
+1

**Pull request merged**